samples/nrf9160/fmfu_smp_svr/prj.conf

```diff
@@ -25,9 +23,21 @@ CONFIG_STATS_NAMES=y
 CONFIG_MCUMGR_CMD_OS_MGMT=y
 CONFIG_MCUMGR_CMD_STAT_MGMT=y
 
+CONFIG_LIBMODEM=y
+#CONFIG_NRF_SOCKETS_BUILD_FROM_SRC=y
+CONFIG_MGMT_MFU=y
+CONFIG_AT_CMD=y
+CONFIG_AT_CMD_PARSER=y
+CONFIG_MODEM_INFO=y
+
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y 
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+
 # Add RTT (Might introduce bugs/crashes)
-#CONFIG_HAS_SEGGER_RTT=y
-#CONFIG_USE_SEGGER_RTT=y
-#CONFIG_LOG_BACKEND_RTT=y
-#CONFIG_LOG=y
-#CONFIG_LOG_BUFFER_SIZE=1024
+CONFIG_USE_SEGGER_RTT=y
```

Comment:
Depends on how extensive you are logging. But I think we can leave this default as off and document that you need to use RTT to log since the serial line is busy.